eva_sub_cli/etc/eva_schema.json

```diff
@@ -165,6 +165,14 @@
           "referenceGenome": {
             "type": "string",
             "description": "Reference against which the analysis was performed. ENA or NCBI accession (starting with GCA) accepted."
+          },
+           "assemblyFasta": {
```

Comment:
In the spreadsheet we request "Reference" and state it can be "GCA accession for assemblies or associated INSDC accession for sequences", and then ask for "Assembly Fasta Path" which sounds like it should be for an assembly only.
Meanwhile here in the JSON we request "referenceGenome" and say it has to start with GCA.

I think we should try to be consistent with our terminology (and also whether we state we accept single INSDC sequences or not...)

Reply:
updated